bin/cqlsh.py

```diff
@@ -2421,7 +2421,7 @@ def read_options(cmdlineargs, environment):
             print("\nWarning: Password is found in an insecure cqlshrc file. The file is owned or readable by other users on the system.",
                   end='', file=sys.stderr)
         print("\nNotice: Credentials in the cqlshrc file is deprecated and will be ignored in the future."
-              "\nPlease use a credentials file to specify the username and password.\n", file=sys.stderr)
+              "\nPlease use a credentials file to specify the username and password.\n")
```

Comment:
I don't think we should change this

Errors and warnings go to stderr

Reply:
Ok, I agree. Printing notice to stdout is also bad, although printing notice to stderr is not good either.
Then I’ll propose another fix, with an expanded notice description